frontendResourceBundles/src/main/resources/GitMacheteBundle.properties

```
@@ -126,7 +126,9 @@ action.GitMachete.BaseSlideInBranchBelowAction.task-title=Sliding in
 
 action.GitMachete.BaseSlideOutBranchAction.description=Slide out ''{0}''
 action.GitMachete.BaseSlideOutBranchAction.notification.title.slide-out-fail=Slide out of <b>{0}</b> failed
-action.GitMachete.BaseSlideOutBranchAction.notification.title.slide-out-success=Branch <b>{0}</b> slid out
+action.GitMachete.BaseSlideOutBranchAction.notification.title.slide-out-success.of-current=Branch has been <b>{0}</b> slid out but not deleted as it is currently checked out
```

Erm... rather (here and below):
```suggestion
action.GitMachete.BaseSlideOutBranchAction.notification.title.slide-out-success.of-current=Branch <b>{0}</b> has been slid out but not deleted as it is currently checked out
```